test/integrations/destinations/emarsys/processor/data.ts

```diff
@@ -1377,4 +1377,121 @@ export const data = [
       },
     },
   },
+  {
```

Comment:
There's no need to write component test case for covering scenario of `eventsMapping` being null, It can be done by simple unit test around `deduceEventId`

Reply:
I see we are failing the event itself, my bad - good with the test case